emmaadesile/shopmate-mobile
# Setup async storage for app

### What does this PR do?

It sets up Async storage for the app

### Description of Task to be completed?

- [x] Check the auth status of a user when they launch the app
- [x] Save `userToken` in `AsynStorage` after signup or login

### How should this be manually tested?

- `git fetch ft-async-storage`
- `yarn install`
- `expo start`
- Run iOS emulator
- Login or signup to the app
- If you're launching the app for the first time, you should see the app intro screen. If not, you are taken to the auth page

### Any background context you want to provide?

None

### Screenshots (if appropriate)

None

### Questions:

None
